src/components/DynamicGrid/DynamicGridColumn.tsx

```diff
@@ -0,0 +1,24 @@
+import { Grid, GridColProps } from '@mantine/core';
+
+import { useWindowSize } from '@/windowmanagement/Window/hooks';
+
+import { useDynamicGridProvider } from './hooks';
+
+export function DynamicGridColumn({ children, span, ...props }: GridColProps) {
+  const { width } = useWindowSize();
```

Comment:
Do we want to make this more general? Now in the missions panel for example, we dont actually have the width of the whole window. It works anyway but the actual minChildSize is smaller than set in the prop, because we have this margin next to it. 

An alternative is to pass the "parentWidth" to the provider, then we can determine ourselves if it should use the useWindowSize hook or something else